lib/rouge/lexers/javascript.rb

```diff
@@ -138,7 +138,6 @@ def self.id_regex
 
       state :root do
         rule /\A\s*#!.*?\n/m, Comment::Preproc, :statement
-        rule /\n/, Text, :statement
```

Comment:
Removing this doesn't seem to break any of the other samples in the visual samples, but fixes the multiline templates.

This doesn't deal with proper ASI anyway.